commcare_connect/organization/decorators.py

```diff
@@ -52,7 +55,7 @@ def _inner(request, *args, **kwargs):
             return HttpResponseRedirect("{}?next={}".format(reverse("account_login"), request.path))
 
         if not permission_test_function(request):
-            raise Http404
+            raise PermissionDenied()
```

Comment:
I think we typically 404 in these situations to not reveal if the org exists, and it was likely intentional

Reply:
> all other permissions decorators that also raise a 404 error ... to keep things consistent, I went ahead with this update

Can you explain a bit more? It sounds like you are saying others use 404, if so, how does this keep things consistent?